File: trunk/csv2code/src/shells/generate/lua/activity_data.lua
--
-- 活动开启时间表.csv
-- 编号 = id
-- 名称 = name
-- 排序 = order
-- 开启等级 = openLv
-- 开放时间 = openTime
-- 开放时间显示 = timeShow
-- 每日免费次数 = freeNum
-- 描述 = describe
-- 玩法描述 = playDsc
-- 是否推送 = open

local data = {
    [1000] = {["id"] = 1000, ["name"] = "世界首领", ["order"] = 2, ["openLv"] = 22, ["openTime"] = {"11:30:00,12:30:00||", "18:00:00,18:30:00||"}, ["timeShow"] = "每天中午12点和晚上8点开启", ["freeNum"] = 1, ["describe"] = "可以获得大量铜钱、元宝、稀有道具奖励", ["playDsc"] = "每天中午12点和晚上8点开启", ["open"] = 1},
    [1001] = {["id"] = 1001, ["name"] = "每日补体力", ["order"] = 2, ["openLv"] = 0, ["openTime"] = {"12:00:00,13:00:00||", "18:00:00,19:00:01||"}, ["timeShow"] = "每天中午12点和晚上6点开启", ["freeNum"] = 2, ["describe"] = "可以获得体力", ["playDsc"] = "每天中午12点和晚上6点开启", ["open"] = 0}
}

return data

File: trunk/csv2code/src/shells/generate/java/ActivityData.java
package com.gamejelly.gong2.config.data;

import com.gamejelly.gong2.config.data.base.LData;
import com.gamejelly.gong2.config.data.base.LMap;

/**
 * 活动开启时间表.csv
 * 编号 = id
 * 名称 = name
 * 开启等级 = openLv
 * 开放时间 = openTime
 * 每日免费次数 = freeNum
 * 是否推送 = open
 */
public class ActivityData extends LData {
    public static final LMap data = new LMap();

    static {
        init0();
        data.setFrozen(true);
    }

    private static void init0() {
data.map(
    1000, map("id", 1000, "name", "世界首领", "openLv", 22, "openTime", list("11:30:00,12:30:00||", "18:00:00,18:30:00||"), "freeNum", 1, "open", 1),
    1001, map("id", 1001, "name", "每日补体力", "openLv", 0, "openTime", list("12:00:00,13:00:00||", "18:00:00,19:00:01||"), "freeNum", 2, "open", 0)
);
    }

}